src/imageLoader/wadors/metaDataManager.js

```diff
@@ -2,6 +2,66 @@ import imageIdToURI from '../imageIdToURI.js';
 
 let metadataByImageURI = [];
 
+function getValue(tag, justElement = true) {
+  if (tag) {
+    if (tag.Value) {
+      if (tag.Value[0] && justElement) {
+        return tag.Value[0];
+      }
+
+      return tag.Value;
+    }
+  }
+
+  return tag;
+}
+
+function combineFrameInstance(frame, instance) {
```

Comment:
Please refactor this in utilities or another file

Reply:
ok